### PR TITLE
fix: expired cookie error on captcha

### PR DESCRIPTION
**Pull request by ntheile**

### CODE REVIEW

File: components/captcha-challenge.tsx [hunk 1]
```diff
@@ -77,6 +78,11 @@ const CaptchaChallenge: React.FC<{ phoneNumber: string }> = ({ phoneNumber }) =>
 
   useEffect(() => {
     const initCaptcha = async () => {
+      try {
+        await clearCookies()
+      } catch (e) {
```
Comment (nicolasburtey) [inline components/captcha-challenge.tsx]: We should not use one letter variable name generally

File: components/captcha-challenge.tsx [hunk 2]
```diff
@@ -77,6 +78,11 @@ const CaptchaChallenge: React.FC<{ phoneNumber: string }> = ({ phoneNumber }) =>
...
+        console.debug("[Init captcha err]:", "no cookies to clear")
```
Comment (nicolasburtey) [inline components/captcha-challenge.tsx]: No need for the comma here

File: components/captcha-challenge.tsx [hunk 3]
```diff
@@ -128,17 +134,38 @@ const CaptchaChallenge: React.FC<{ phoneNumber: string }> = ({ phoneNumber }) =>
   const isLoading = captchaState.status === "loading" || createLoading || requestLoading
   const hasError = !isLoading && captchaState.status === "error"
 
+  const resetPage = () => {
+    clearCookies().finally(() => {
```
Comment (nicolasburtey) [inline components/captcha-challenge.tsx:134]: Let's use async/await to he consistent across our codebases

File: components/captcha-challenge.tsx [hunk 4]
```diff
@@ -77,6 +78,11 @@ const CaptchaChallenge: React.FC<{ phoneNumber: string }> = ({ phoneNumber }) =>
 
   useEffect(() => {
     const initCaptcha = async () => {
+      try {
+        await clearCookies()
+      } catch (e) {
+        console.debug("[Init captcha err]:", "no cookies to clear")
+      }
```
Comment (dolcalmi) [inline components/captcha-challenge.tsx:85]: why not include it in the other trycatch block?

**Review by dolcalmi — CHANGES_REQUESTED** [review]

File: components/captcha-challenge.tsx (hunk 4 above)
Comment (ntheile) [reply components/captcha-challenge.tsx:85]: i think it returns a 500 if no cookies and i still want it to proceed to the captcha code. not sure if the browser fetch throws on a 500 or not

Comment (ntheile) [reply components/captcha-challenge.tsx:85]: actually just looked it up, it does not throw on 500s (maybe its axios that does that).  i will move it into the main try

**Review by nicolasburtey — COMMENTED** [review]
Few comments

File: components/captcha-challenge.tsx (hunk 3 above)
Comment (ntheile) [reply components/captcha-challenge.tsx]: yep got fetch mixed up with axios. too many client http libs to keep track of lol

**Review by dolcalmi — APPROVED** [review]

**Pull request merged**